doc/user/data/disaster_recovery/system_environment.yml

```diff
@@ -0,0 +1,27 @@
+columns:
+  - column: "Failure Type"
+  - column: "Impact"
+rows:
+  - Failure Type: "**Single Availability Zone (AZ)**"
+    Impact: |
+      - Connection issues using single-AZ Privatelink and sources/sinks.
+
+      - Brief `pgwire` and `https` connection drops as network rebalances.
+
+  - Failure Type: "**Two Availability Zones**"
+    Impact: |
+      - Temporary issues with cluster provisioning.
+      - Temporary issues with Console access.
+
+  - Failure Type: "**Three or More Availability Zones**"
+    Impact: |
+      - Partial to no access to the database.
```

Comment:
Wasn't sure if the "limited" in "limited to no database access" meant:
-  only database access is lost (but no data loss, etc) or
- partial loss